pkg/log/exporter.go

```diff
@@ -58,6 +61,7 @@ func (ex *exporter) Fire(entry *logrus.Entry) error {
 	ex.wg.Add(1)
 
 	go func(entry *logrus.Entry) {
+		<-ex.waitForRegistrationCh
```

Comment:
this channel to block pushing messages is very barebones. it feels like it's a castai.Client responsibility?

```suggestion
		ex.client.WaitForLogsToBeReady(ctx)
```